ui/src/containers/ClusterMonitoring.js

```diff
@@ -222,7 +222,7 @@ const ClusterMonitoring = props => {
               </TooltipContent>
             }
           >
-            <QuestionMarkIcon className="fas fa-question-circle" />
+            <QuestionMarkIcon className="fas fa-info-circle" />
```

Comment:
Not related to this PR (and can't be fixed here, I supposed it's coming from core-ui), but the naming seems poor to me: we call `QuestionMarkIcon` something that's not always a question mark icon…

Reply:
Forgot to change the name `QuestionMarkIcon` to `InformationMarkIcon`, my bad.
Sure, it's not related to this PR, but it's a tiny change in order to keep the consistency for the icons. 
And just FYI, the icon we import directly from the font-awsome library ;)